openstack/identity/v3/tokens/requests.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/gophercloud/gophercloud/v2"
 )
 
+const defaultAccessRulesVersion = "1"
```

Comment:
it's better to set it to `1.0`, see https://github.com/openstack/keystone/blob/bd178ee91f92f50398cf940b1db3eb2157ed7fd3/keystone/models/token_model.py#L33